apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -813,12 +831,22 @@ export class LemonadePipeline implements BasePipeline {
         });
       }
 
+      const ticketPCDs = await Promise.all(
+        tickets.map((t) => EdDSATicketPCDPackage.serialize(t))
+      );
+
+      if (this.definition.options.enablePODTickets) {
```

Comment:
potentially worth moving the declaration and initialization of `podTickets` into this `if` statement